full/Dockerfile

```diff
@@ -275,6 +275,21 @@ ENV PATH=$PATH:$HOME/.cargo/bin
 
 RUN bash -lc "cargo install cargo-watch cargo-edit cargo-tree"
 
+### Docker ###
+LABEL dazzle/layer=docker
+LABEL dazzle/test=tests/tool-docker.yaml
+USER root
```

Comment:
Q: Wouldn't it make more sense to install it as `USER gitpod` with `sudo`?

Reply:
no, why? I find doing `USER root` once more simple than prefixing every command with `sudo`.

Reply:
In the past this has caused users to shoot themselves in the foot, by doing `USER root` and then accidentally creating files that were inaccessible to user `gitpod` (leading to annoying problems in their workspace).

But, since `USER root` is already commonplace in `workspace-full`, adding one more is probably okay (because `workspace-full` is arguably "less user-facing", thus incurs less risk of trapping users).

Still, a return to `USER gitpod` at the bottom would be nice (even though the layer below re-does `USER root`, I think the Dazzle layers are supposed to be runnable in any order).